ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/monitor_telemetry_postcard.schema.yml

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  monitor_telemetry_postcard:
```

Comment:
how do i set `marker` option?
```
coi212(config-tele-postcard-policy)#?
  disabled  Enable or disable the postcard telemetry feature
  ingress   Configure ingress paramters
  marker    Configure marker mode parameters  <----- THIS
  profile   Configure postcard telemetry profile
  sample    Configure sampling feature
  ```

Reply:
I am not able to see marker in running config when configured.

Reply:
s1-leaf1(config-tele-postcard-policy)#marker vxlan header Word 0 bit 10 
s1-leaf1(config-tele-postcard-policy)#show active 
monitor telemetry postcard policy
s1-leaf1(config-tele-postcard-policy)#

Reply:
Fixed.